README.rst

```diff
@@ -106,22 +106,36 @@ Example:
 
    $ mkdir test && cd test && git init
    Initialized empty Git repository in /tmp/test/.git/
-   $ echo "if True: print('hi')\n\nif False: print('there')" | tee test.py
-   if True: print('hi')
 
+   $ echo "if True: print('hi')\nprint()\nif False: print('there')" | tee test.py
```

Comment:
For readability, i don't think you should include the code used to generate the examples as they themselves distract away from the actual example. I can't nest backticks in the suggestion block so i'll just make this a comment instead:

### our_file.py at commit hash XXXXXXXX
```
if True: print('hi')

if False: print('there')
```
### An uncommitted change to the our_file.py:
<pre><code>if True: print('<b>changed</b>')

if False: print('there')
</code></pre>

### Show changes to the file(s) darker would make
Note, the `if False: print('there')` is omitted as it was not modified!
```
$ darker --diff ./our_file.py -r XXXXXXXX
> --- test.py
> +++ test.py
> @@ -1,3 +1,4 @@
> -if True: print('changed')
> +if True:
> +    print("changed")
```
You can also specify all files in the current directory:
all files under the current directory
```
$ darker --diff . -r XXXXXXXX
```

### Black the file(s) with darker!
```
$ darker ./our_file.py  -r XXXXXXXX
```
or 
```
$ darker .  -r XXXXXXXX
```
Output our blacked file:
Note, the `if False: print('there')` is not blacked as it was not been modified!
```
$ cat ./our_file.py
> if True: 
>     print('changed')
> if False: print('there')
```

Reply:
@akaihola hopefully you find these suggestions useful, if you'd like i can submit them in another pr

Reply:
Yes, this is very useful! I actually wanted to combine your suggestions immediately wtih some of my own ideas – what do you think of the example now?

Reply:
One modification to your suggestion was that I didn't want to introduce the `-r` switch in the very first example – for local interactive use I expect the default comparison to `HEAD` to be the sane default most often. It's in the CI environment where `--revision` becomes especially handy.